tests/nnstreamer_filter_python3/runTest.sh

```diff
@@ -37,23 +37,31 @@ if [[ -d $PATH_TO_PLUGIN ]]; then
     fi
 else
     echo "No build directory"
-    report
-    exit
+    # Check if module is installed.
+    INSTALL_PATHS=$(python -c 'import sys; print ("\n".join(sys.path))')
```

Comment:
```suggestion
    INSTALL_PATHS=$(python3 -c 'import sys; print ("\n".join(sys.path))')
```

Or.. How about import module?
```
    if python3 -c "import nnstreamer_python" &> /dev/null; then
        echo 'nnstreamer_python module exists.'
    else
        echo 'nnstreamer_python module does not exist'
        report
        exit
    fi
```